config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -5,7 +5,7 @@ periodics:
   annotations:
     fork-per-release: "true"
     fork-per-release-periodic-interval: 1h 2h 6h 24h
-    testgrid-dashboards: sig-release-master-blocking, sig-node-kubelet
+    testgrid-dashboards: sig-release-master-blocking, sig-node-kubelet, sig-node-blocking
```

Comment:
what about putting it in "sig-node-kubelet-blocking"?
thatway each area(kubelet, containerd, cri, etc) can have a blocking and a "nonblocking" (jobs that are maintained and working as intended versus jobs that need attention).